README.md

```diff
@@ -30,29 +30,37 @@ See also:
 
 Use QGIS to generate input for **routing**, **isochrones** and **matrix calculations** powered by ORS.
 
-It offers either a GUI in the Web menu and toolbar of QGIS to interactively use the API's from the map canvas.
+You'll have to create an openrouteservice account and get a free API key first: <https://openrouteservice.org/sign-up>.
+After you have received your key, add it to the default `openrouteservice` provider via `Web` ► `ORS Tools` ►
+`Provider Settings` or click the settings button in the ORS Tools dialog.
 
-For batch operations you can find an ORS toolbox in the Processing toolbox.
+The plugin offers either a GUI in the `Web` menu and toolbar of QGIS to interactively use the ORS API
+from the map canvas.
 
-### Customization
+For batch operations you can find an `ORS Tools` folder in the Processing Toolbox.
 
-You'll have to get an API key first: <https://openrouteservice.org/sign-up>.
+### Customization
 
-Additionally, you can register your other providers, e.g. if you're hosting a custom ORS backend.
+Additionally, you can register other ORS providers, e.g. if you're hosting a custom ORS backend.
 
-Configuration takes place either from the Web menu entry *ORS Tools* ► *Provider settings*. Or from *Config* button in the GUI.
+Configuration takes place either from the Web menu entry *ORS Tools* ► *Provider settings*. Or from the *Config* button
+in the GUI.
 
 ## Getting Started
 
 ### Prerequisites
 
-QGIS version: min. **v3.0**
+QGIS version: **v3.4** or above
 
 API key: https://openrouteservice.org/sign-up/
 
 ### Installation
 
-Either from QGIS plugin repository or manually:
+In the QGIS menu bar click `Plugins` ► `Manage and Install Plugins...`.
+
+Then search for `openrouteservice` and install `ORS Tools`.
+
+Or install the plugin manually:
```

Comment:
Replace the "Or" with "Alternatively, you can " or something else

Reply:
done